server/src/MessageHandler.ts

```diff
@@ -226,6 +270,51 @@ export async function handleDocumentFormatting(
   ])
 }
 
+export function handleHoverRequest(
+  documents: TextDocuments<TextDocument>,
+  params: HoverParams,
+): Hover | undefined {
+  const textDocument = params.textDocument
+  const position = params.position
+
+  const document = documents.get(textDocument.uri)
+
+  if (!document) {
+    return
+  }
+
+  const lines = convertDocumentTextToTrimmedLineArray(document)
+  const word = getWordAtPosition(document, position)
+
+  if (word === '') {
+    return
+  }
+
+  const foundBlock = getModelOrEnumBlock(word, lines)
+  if (!foundBlock) {
+    return
+  }
+
+  const commentLine = foundBlock.start.line - 1
+  const docComments = document.getText({
+    start: { line: commentLine, character: 0 },
+    end: { line: commentLine, character: 9999999 },
```

Comment:
Just curious about this usage of `9999999`, is there any significance to this or is it just a large enough number? Can this be replaced with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER?

Reply:
I used that because I want to get the whole line and don't know the line length all the time. In the documentation it states: `If the character value is greater than the line length it defaults back to the line length. If a line number is negative, it defaults to 0.` That's why I used that number.

But I agree, it'd be a better idea to replace this with MAX_SAFE_INTEGER.